pandavro/__init__.py

```diff
@@ -94,9 +127,34 @@ def __schema_infer(df, times_as_micros):
     return schema
 
 
-def __file_to_dataframe(f, schema, **kwargs):
+def __file_to_dataframe(f, schema, na_dtypes=False, **kwargs):
     reader = fastavro.reader(f, reader_schema=schema)
-    return pd.DataFrame.from_records(list(reader), **kwargs)
+    df = pd.DataFrame.from_records(list(reader), **kwargs)
+
+    def _filter(typelist):
+        # It's a string, we return it directly
+        if type(typelist) == str:
+            return typelist
+        # If a logical type dict, it has a type attribute. Return None as we don't touch logical types
+        elif type(typelist) == dict:
+            return None
+            return _filter(typelist["type"])
+        else:
+            l = [t for t in typelist if t != "null"]
+            if len(l) > 1:
+                raise ValueError(f"More items in list than 1. {l}")
```

Comment:
This line was failed in Python 2.7, 3.5. f-strings was introduced on 3.6.

```
==================================== ERRORS ====================================
___________________ ERROR collecting tests/pandavro_test.py ____________________
../../../virtualenv/python3.5.6/lib/python3.5/site-packages/_pytest/python.py:445: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
../../../virtualenv/python3.5.6/lib/python3.5/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:968: in _find_and_load
    ???
<frozen importlib._bootstrap>:957: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:664: in _load_unlocked
    ???
<frozen importlib._bootstrap>:634: in _load_backward_compatible
    ???
../../../virtualenv/python3.5.6/lib/python3.5/site-packages/_pytest/assertion/rewrite.py:308: in load_module
    six.exec_(co, mod.__dict__)
tests/pandavro_test.py:4: in <module>
    import pandavro as pdx
E     File "/home/travis/build/ynqa/pandavro/pandavro/__init__.py", line 145
E       raise ValueError(f"More items in list than 1. {l}")
E                                                        ^
E   SyntaxError: invalid syntax
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.51 seconds ============================
```

Reply:
Removed this f-string